apps/rewards/app/app-state-reducer.js

```diff
@@ -58,8 +61,26 @@ function appStateReducer(state) {
         unit: 'USD',
       },
     ]
-    state.myMetrics = state.myMetrics || []
+    let myMetric = calculateMyRewardsSummary(state.rewards, state.balances, convertRates)
+    state.myMetrics = [
+      {
+        name: 'Unclaimed rewards',
+        value: myMetric[0].toString(),
+        unit: 'USD',
+      },
+      {
+        name: 'Rewards obtained this year',
+        value: myMetric[1].toString(),
+        unit: 'USD',
+      },
+      {
+        name: 'All time rewards obtained',
+        value: myMetric[2].toString(),
+        unit: 'USD',
+      },
+    ]
     state.amountTokens = state.amountTokens || []
+    console.log('end recuders:', state)
```

Comment:
Do you think it's useful to leave this in for now? Do we need an `eslint-disable-line` comment?